docs/detections/prebuilt-rules/rule-details/component-object-model-hijacking.asciidoc

```diff
@@ -72,3 +75,10 @@ user.domain != "NT AUTHORITY")
 ** Name: Event Triggered Execution
 ** ID: T1546
 ** Reference URL: https://attack.mitre.org/techniques/T1546/
+
+[[component-object-model-hijacking-history]]
+==== Rule version history
+
+Version 3 (7.12.0 release)::
+* Formatting only
```

Comment:
Looks like the indices and look-back time was modified for this rule, but we're saying the update consists of formatting changes only. Is that intentional?

Reply:
i believe if the rule query was not modified, then it'll go down as formatting only.